protected/views/layouts/new_main.php

```diff
@@ -134,6 +134,7 @@
                             <ul class="dropdown-menu">
                                 <li><a href="/site/help">Help</a></li>
                                 <li><a href="/site/faq">FAQ</a></li>
+                <li><a href="/site/guide">Guidelines</a></li>
```

Comment:
Should correct the indentation for this line so it is consistent with the rest of the code.